### PR TITLE
SPORTS-54 Re-arrange the folder structure

### PR DESCRIPTION
**Pull request by Ishanliv99**
Previously: 
  - Improper file names.
  - Files/Components not located in appropriate folders.

Now:
  - Correct above.

### CODE REVIEW

File: frontend/src/components/Routes.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import { Router, Route, Switch } from 'react-router';
 
-import User from './user';
 import Login from './auth';
 import Tree from './tournament/tree';
+import User from './tournament/fixtureOverview';
```
Comment (aviskarkc10) [inline frontend/src/components/Routes.js]: Change the component name to FixtureOverview or something relevant

File: frontend/src/components/tournament/fixtureOverview/index.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 
-import Fixture from '../fixture';
-import Dates from '../commons/date';
-import Title from '../commons/title';
-import Header from '../commons/header';
+import Dates from '../../commons/dates';
+import FixtureCard from './fixture-card';
+import Title from '../../commons/title';
```
Comment (aviskarkc10) [inline frontend/src/components/tournament/fixtureOverview/index.js]: length rule

File: frontend/src/components/Routes.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 
 import * as routes from '../constants/routes';
```
Comment (arun-tamang) [inline frontend/src/components/Routes.js]: import constants before components.

File: frontend/src/components/tournament/fixtureOverview/index.js [hunk 4]
```diff
@@ -1,38 +1,38 @@
 import React from 'react';
 
-import Fixture from '../fixture';
-import Dates from '../commons/date';
-import Title from '../commons/title';
-import Header from '../commons/header';
+import Dates from '../../commons/dates';
+import Title from '../../commons/title';
+import FixtureCard from './fixture-card';
```
Comment (aviskarkc10) [inline frontend/src/components/tournament/fixtureOverview/index.js]: use camelCase for foldernames

**Pull request merged**